packages/react-tooltip/src/components/Tooltip/useTooltip.tsx

```diff
@@ -15,7 +15,7 @@ import type { TooltipProps, TooltipState, TooltipTriggerProps } from './Tooltip.
 
 // Style values that are required for popper to properly position the tooltip
 const tooltipBorderRadius = 4; // Update the root's borderRadius in useTooltipStyles.ts if this changes
-const arrowHeight = 6; // Update the arrow's width/height in useTooltipStyles.ts if this changes
+const arrowHeight = 6; // Update arrowHeight in useTooltipStyles.ts if this changes
```

Comment:
Not blocking for this PR: Is this a case for maybe exporting out of one file and importing into another while not exporting from the package itself so we don't have duplicate information?

Reply:
I'll make that change in a follow-up PR

Reply:
#21204